### PR TITLE
Adding new property isBootstrapCertificate to CSES

### PR DESCRIPTION
**Pull request by avilgoelms**
# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

<a href="?expand=1&template=sdk_configuration_template.md">Click here</a> to open a PR for only SDK configuration.

### CODE REVIEW

**Review by haagha — CHANGES_REQUESTED** [review]
Please run the prettier check and target this to main branch.

File: specification/compute/resource-manager/Microsoft.Compute/CloudserviceRP/stable/2024-11-04/cloudService.json [hunk 1]
```diff
@@ -2014,6 +2014,10 @@
         "certificateUrl": {
           "description": "This is the URL of a certificate that has been uploaded to Key Vault as a secret.",
           "type": "string"
+        },
+        "isBootstrapCertificate": {
+          "description": "Flag indicating if the certificate provided is a bootstrap certificate to be used by the Key Vault Extension to fetch the remaining certificates. ",
+          "type": "bool"
```
Comment (haagha) [inline specification/compute/resource-manager/Microsoft.Compute/CloudserviceRP/stable/2024-11-04/cloudService.json:2020]:
```suggestion
          "type": "boolean"
```

**Review by haagha — CHANGES_REQUESTED** [review]
Change bool to boolean, also please run the prettier check on example file and target this to main branch.

**Comment by grizzlytheodore** [comment]
Please address haider comment. That should resolve most of the failing CI failures.
But we want to have lintdiff, modelvalidation, prettiercheck, and semanticvalidation all passing

**Review by grizzlytheodore — APPROVED** [review]

**Pull request merged**